src/core/src/main.rs

```diff
@@ -112,10 +121,10 @@ fn main() {
         .encoder(Box::new(PatternEncoder::new(
             "{d:22.22} {h({l:5.5})} | {({M}:{L}):30.30} | {m}{n}",
         )))
-        .build("log/debug.log")
+        .build(format!("{}{}",config.get_log_dir(), "poet-consensus.log"))
```

Comment:
[Thought]: How about std::path::Path::new(config.get_log_dir()).join("poet-consensus.log").to_str().expect("Unable to get log file path") ?